src/shared/components/ResourceEditor/index.tsx

```diff
@@ -146,23 +140,41 @@ const ResourceEditor: React.FunctionComponent<ResourceEditorProps> = props => {
 
   const handleCancel = () => {
     setStringValue(JSON.stringify(rawData, null, 2));
-    setValid(true);
+    setIsValidJSON(true);
     setEditing(false);
   };
 
+  const handleLintError = useCallback(
+    (errors: LinterIssue[]) => {
+      console.log(errors);
```

Comment:
Is the console.log intentional?

Reply:
No, I'm removing it. (We should consider adding a linter setting to warn about leftover console.logs) 😁